run_analysis.py

```diff
@@ -154,6 +142,22 @@ def analysis(config): #{{{
     config = MpasAnalysisConfigParser()
     config.read(configFileNames)
 
+    # choose the right rendering backend, depending on whether we're displaying
+    # to the screen
+    if not config.getboolean('plot', 'displayToScreen'):
+        mpl.use('Agg')
+    import matplotlib.pyplot as plt
+    # analysis can only be imported after the right MPL renderer is selected
+    from mpas_analysis.ocean.ohc_timeseries import ohc_timeseries
+    from mpas_analysis.ocean.sst_timeseries import sst_timeseries
+    # from mpas_analysis.ocean.nino34_timeseries import nino34_timeseries
+    # from mpas_analysis.ocean.mht_timeseries import mht_timeseries
+    # from mpas_analysis.ocean.moc_timeseries import moc_timeseries
+    from mpas_analysis.ocean.ocean_modelvsobs import ocn_modelvsobs
+
+    from mpas_analysis.sea_ice.timeseries import seaice_timeseries
+    from mpas_analysis.sea_ice.modelvsobs import seaice_modelvsobs
+
```

Comment:
@xylar, sorry about the confusion regarding the idea above that is clearly false.  Sorry about the bad idea.

I do have a point of clarification, however: why move all the module imports into the `__main__`?  It seems like it may be cleaner to have them where they were previously because then when we need to add a new analysis the code modification will be much more centralized to allow easier extension by future users not familiar with the whole system.  I think in this case it may not be the right design solution to separate the import and run code.  Also, I think we should keep `__main__` as simple and as short as possible.  I think if we have to break PEP8 conventions to ensure greater clarity in the code it is worth it.